### ISSUE TITLE
incorrect 'insufficient contrast' error for svg title attribute

### ISSUE THREAD

**Issue by hollsk** [squizlabs/HTML_CodeSniffer#229]
hello squizlabs! ❤️  👋 

A Pa11y user opened an extremely detailed issue about WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail  at https://github.com/pa11y/pa11y/issues/416 

The summary is that a contrast error is returned on an SVG title, which is not ordinarily visible. 

>Error: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1:1. Recommendation: change text colour to #767676.
>   ├── WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
>   ├── html > body > div:nth-child(1) > svg > title
>  └── <title>Invisible title!</title>

I replicated it with this reduced test case:

```
<div style="color:white;background:white;">
	<svg width="10" height="10" viewBox="0 0 10 6">
		<title>Invisible title!</title>
		<circle id = "s1" cx = "10" cy = "10" r = "10" fill = "yellow" stroke = "black" stroke-width = "1"/>
	</svg>
</div>
```

More info in the original Pa11y issue linked above! ☺️
